core/Command/SystemTag/Edit.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * @copyright Copyright (c) 2021, hosting.de, Johannes Leuker <developers@hosting.de>
+ *
+ * @author Johannes Leuker <developers@hosting.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OC\Core\Command\SystemTag;
+
+use OC\Core\Command\Base;
+use OCP\SystemTag\ISystemTagManager;
+use OCP\SystemTag\TagAlreadyExistsException;
+use OCP\SystemTag\TagNotFoundException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Edit extends Base {
+
+	/** @var ISystemTagManager */
+	protected $systemTagManager;
+
+	/**
+	 * @param ISystemTagManager $systemTagManager
+	 */
+	public function __construct(ISystemTagManager $systemTagManager) {
+		$this->systemTagManager = $systemTagManager;
+		parent::__construct();
+	}
+
+	protected function configure() {
+		$this
+			->setName('tag:edit')
+			->setDescription('edit tag attributes')
+			->addArgument(
+				'id',
+				InputOption::VALUE_REQUIRED,
+				'The ID of the tag that should be deleted',
+			)
+			->addOption(
+				'name',
+				null,
+				InputOption::VALUE_OPTIONAL,
+				'sets the \'name\' parameter',
+			)
+			->addOption(
+				'access',
+				null,
+				InputOption::VALUE_OPTIONAL,
+				'sets the access control level (public, restricted, invisible)',
+			);
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$tag_arr = $this->systemTagManager->getTagsByIds($input->getArgument('id'));
+		// returns an array, but we always expect 0 or 1 results
+
+		if ($tag_arr) {
+			$tag = array_values($tag_arr)[0];
```

Comment:
```suggestion
		$tagArray = $this->systemTagManager->getTagsByIds($input->getArgument('id'));
		// returns an array, but we always expect 0 or 1 results

		if ($tagArray) {
			$tag = array_values($tagArray)[0];
```

to stick to https://docs.nextcloud.com/server/latest/developer_manual/getting_started/codingguidelines.html#objects-functions-arrays-variables